### PR TITLE
Task/APPS-3019 — fix scroll on mobile

### PR DESCRIPTION
**Pull request by giuliacampana**
trying out LUK drawer update

https://user-images.githubusercontent.com/32921059/135351243-570256f8-301d-4285-b68e-476e98738c34.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,10 +36,11 @@
     "connected-react-router": "~6.9.0",
     "history": "~4.10.0",
     "immutable": "4.0.0-rc.10",
-    "lattice-ui-kit": "~0.46.0",
+    "lattice-ui-kit": "~0.46.1",
     "lodash": "~4.17.21",
     "loglevel": "~1.7.0",
     "polished": "~4.1.2",
+    "punycode": "^2.1.1",
```
Comment (UnsungHero97) [inline package.json:43]: What is this for and where is it used?

File: src/components/navigation/NavItems.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * @flow
  */
 
-import React, { useState } from 'react';
+import React, { Fragment, useState } from 'react';
```
Comment (UnsungHero97) [inline src/components/navigation/NavItems.js:5]: Are you using `Fragment` anywhere?

File: src/components/navigation/NavItems.js [hunk 3]
```diff
@@ -63,9 +63,10 @@ const NavItemsWrapper = styled.div`
 const PortalOuterWrapper = styled.div`
   -webkit-overflow-scrolling: touch;
   height: 100%;
-  overflow: scroll;
+  overflow-y: scroll;
   position: fixed;
   right: 0;
+  scroll-behavior: smooth;
```
Comment (UnsungHero97) [inline src/components/navigation/NavItems.js:69]: What does this for? It's not very well supported :(

https://caniuse.com/?search=scroll-behavior

File: package.json (hunk 1 above)
Comment (giuliacampana) [reply package.json]: i literally have no idea how or why that's there... i removed it.

File: src/components/navigation/NavItems.js (hunk 2 above)
Comment (giuliacampana) [reply src/components/navigation/NavItems.js]: nope, i removed it—i was using it before but the idea didn't work.

File: src/components/navigation/NavItems.js (hunk 3 above)
Comment (giuliacampana) [reply src/components/navigation/NavItems.js]: ya, you're right :( i removed it. it doesn't seem to actually be affecting scroll behavior on desktop.

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**